proto/src/v1.rs

```diff
@@ -917,6 +917,8 @@ pub enum AuthStep {
     Init2 {
         username: String,
         issue: AuthIssueSession,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        privileged: Option<bool>,
```

Comment:
I think you can make this just `bool` because bool impls default, and bool defaults to false. That way you can have `#[serde(default)]` here instead. That then makes some of your later code a bit easier.

Reply:
Done.